de.gonicus.gonnect.yml

```diff
@@ -0,0 +1,117 @@
+id: de.gonicus.gonnect
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+command: gonnect
+
+modules:
+- name: kerberos
+  subdir: src
+  config-opts:
+    - --localstatedir=/var/lib
+    - --sbindir=${FLATPAK_DEST}/bin
+    - --disable-rpath
+    - --disable-static
+  cleanup:
+    - /bin
+    - /share/et
+    - /share/examples
+  post-install:
+    - install -Dm644 ../krb5.conf -t ${FLATPAK_DEST}/etc/
+  sources:
+    - type: archive
+      url: "https://kerberos.org/dist/krb5/1.21/krb5-1.21.3.tar.gz"
+      sha256: b7a4cd5ead67fb08b980b21abd150ff7217e85ea320c9ed0c6dadd304840ad35
+    - type: file
+      path: "krb5.conf"
+
+- name: pjsip
+  buildsystem: autotools
+  config-opts:
+  - --disable-video
+  - --enable-ext-sound
+  - CFLAGS=-fPIC -DPJ_HAS_IPV6=1
+  sources:
+  - type: git
+    url: https://github.com/pjsip/pjproject.git
+    tag: 2.15.1
+
+- name: openldap
+  config-opts:
+    - --disable-static
+    - --disable-slapd
+    - --disable-slurpd
+    - --disable-bdb
+    - --disable-hdb
+    - --with-tls=openssl
+  cleanup:
+    - /bin
+  sources:
+    - type: archive
+      url: "https://www.openldap.org/software/download/OpenLDAP/openldap-release/openldap-2.6.9.tgz"
+      sha256: 2cb7dc73e9c8340dff0d99357fbaa578abf30cc6619f0521972c555681e6b2ff
+
+- name: libusb
```

Comment:
use the shared modules for this.

Reply:
This is my first flatpak. Could you explain what that means?